docs/4-user-guide/2-zarf-packages/3-the-zarf-init-package.md

```diff
@@ -4,36 +4,41 @@ sidebar_position: 3
 | container-registry      | Adds a container registry service&mdash;[docker registry](https://docs.docker.com/registry/)&mdash;into the cluster. |
 
 ## Additional Components
 
-In addition to those that are always installed, Zarf's optional components provide additional functionality and can be enabled as & when you need them.
+In addition to required components, Zarf also offers optional components that provide additional functionality and can be enabled as needed. 
 
-These optional components for the init package are listed below along with the "magic strings" you pass to `zarf init --components` to pull them in:
+Below are the optional components available for the init package, along with their respective "magic strings" that can be passed to `zarf init --components` to pull them in:
 
-| components   | Description                                                                                                                                                       |
+| Components   | Description                                                                                                                                                       |
 | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| k3s          | REQUIRES ROOT. Installs a lightweight Kubernetes Cluster on the local host&mdash;[k3s](https://k3s.io/)&mdash;and configures it to start up on boot.                             |
-| logging      | Adds a log monitoring stack&mdash;[promtail / loki / graphana (a.k.a. PLG)](https://github.com/grafana/loki)&mdash;into the cluster.                              |
+| k3s          | REQUIRES ROOT. Installs a lightweight Kubernetes Cluster on the local host&mdash;[K3s](https://k3s.io/)&mdash;and configures it to start up on boot.                             |
+| logging      | Adds a log monitoring stack&mdash;[promtail/loki/graphana (aka PLG)](https://github.com/grafana/loki)&mdash;into the cluster.                              |
 | git-server   | Adds a [GitOps](https://www.cloudbees.com/gitops/what-is-gitops)-compatible source control service&mdash;[Gitea](https://gitea.io/en-us/)&mdash;into the cluster. |
 
-There are two ways to deploy optional components, you can either pass a comma separated list of components to the `--components` flag such as `zarf init --components k3s,git-server --confirm` or you can exclude the flags and say yes/no as each optional component gets prompted to you.
+There are two ways to deploy optional components. Firstly, you can provide a comma-separated list of components to the `--components` flag, such as `zarf init --components k3s,git-server --confirm`. Alternatively, you can choose to exclude the flags and respond with a yes or no for each optional component when prompted
```

Comment:
```suggestion
There are two ways to deploy optional components. Firstly, you can provide a comma-separated list of components to the `--components` flag, such as `zarf init --components k3s,git-server --confirm`. Alternatively, you can choose to exclude the `--components` and `--confirm` flags and respond with a yes or no for each optional component when prompted.
```
Might be good to clarify that both flags need to be dropped (initing with --confirm alone will deploy with no optional components and won't prompt)